EventHubs/tests/test_eventhub_logs.py

```diff
@@ -24,24 +24,21 @@ def test_pipeline(self):
         self.assertTrue(self.resource_group_exists(self.RESOURCE_GROUP_NAME))
         self.table_service = self.get_table_service()
```

Comment:
remove get_table_service and make this similar to metrics test_pipeline function

Reply:
add
 self.successful_sent_message = 'Sent all data to Sumo. Exit now.'
 self.expected_resource_count = 7

Reply:
updated